Remotion/Web/Development.Analyzers/IsExternalInit.cs

```diff
@@ -1,4 +1,4 @@
 ﻿namespace System.Runtime.CompilerServices
 {
   internal static class IsExternalInit {}
-}
\ No newline at end of file
+}
```

Comment:
do we need this or can we reset those changes?

Reply:
Didnt build without it and now i cant build without rider adding the newline back again. Suspicious. How should I continue here?

Reply:
sounds very strange to me. maybe apply holy water? *SCNR*

live look?

Reply:
Well, it doesnt matter: the style analyzer needs single newline at end of file, so he gets it. If that rule is removed at any point, my pc might need an exorcism.